src/app/app.component.ts

```diff
@@ -1,10 +1,35 @@
 import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
```

Comment:
Create a new component for this, don't use the AppComponent!

Reply:
AppComponent's HTML should only have the <router-outlet></router-outlet> tag, create a new component (with the cli `ng generate component screens/component-name`), put all the html and ts that you already created in this component :)